packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -193,8 +193,15 @@ export function PatternCategoryPreviews( {
 						isDraggable
 						showTitlesAsTooltip={ showTitlesAsTooltip }
 						patternFilter={ patternSourceFilter }
-						pagingProps={ pagingProps }
 					/>
+					{ pagingProps && (
```

Comment:
Unless critical to this PR, then I'd suggest we take the changes that move Pagination and put in a separate PR.